app/models/route_location.rb

```diff
@@ -1,4 +1,5 @@
 class RouteLocation < ApplicationRecord
+  validates_presence_of location_id:, route_id:
```

Comment:
```suggestion
  validates_presence_of :location_id:, :route_id
```

Reply:
@joshisplutar Not sure if this is related but should be updated.

Reply:
corrected and getting same error

Reply:
After you restarted ;)